local_server/local-server-services/README.md

```diff
@@ -1,38 +1,55 @@
 
-**Note:** Local Server is not supported on MacOS
+![Image of local server services](LocalServerServices.png)
 
-![](LocalServerServices.png)
+## Use case
+
+For executing offline geoprocessing tasks in your ArcGIS Runtime apps via an offline (local) server.
 
 ## How to use the sample
 
-Choose an option from the dropdown control to filter packages by service type. Then click the Open button to choose a package. Finally, click the Start button to start the service. The service's status will be displayed in the center log.
+Click `Start Local Server` to start the Local Server. Click `Stop Local Server` to stop the Local Server.
+
+The `Map Service` combo box lets you to pick a local service that is available.
+
+After browsing for the desired file, click `Start Service` to start the selected service.
 
-To stop a service, select it from the Running Services list and click the Stop Service button. To go to the service's URL in your default web browser, select it and click the Go to URL button.
+When the running service's url appears, select it and click `Open Url`. To stop this running service, click `Stop Service`.
 
 ## How it works
 
-To start a `LocalService`:
+To start a `LocalServer` and attach a `LocalService`:
+
+1. Create and run a local server with `LocalServer.instance`.
+2. Start the server asynchronously with `Server.startAsync()`.
+3. Wait for server to be in the  `LocalServerStatus.STARTED` state.
+   * Callbacks attached to `Server.addStatusChangedListener()` will invoke whenever the status of the local server has changed.
+4. Create and run a local service. Here is an example of running a `LocalMapService`:
+    1. Instantiate `LocalMapService(Url)` to create a local map service with the given URL path to map package (`mpk` or `mpkx` file).
+    2. Start the job with `LocalMapServic.estartAsync()`.
+       * The service is added to the `LocalServer` automatically.
+
+To stop a `LocalServer` and any attached `LocalServices`:
 
-1. Start the local server.
-    * `LocalServer.INSTANCE` creates a local server
-    * `server.startAsync()` starts the server asynchronously
-2. Wait for server to be in th. `LocalServerStatus.STARTED` state.
-    * `Server.addStatusChangedListener()` fires whenever the running status of the local server
-changes
-3. Create and run a local service.
-    * `new LocalMapService(Url)` creates a local map service with the given url path to mpk file
-    * `service.startAsync()` starts the service asynchronously
-4. Stop the service with `service.stopAsync().`
+1. If required, you can retrieve a list of all running services with `LocalServer.INSTANCE.getServices()`.
+2. Stop the services asynchronously using `LocalService.stopAsync()`.
+3. Use `LocalServer.INSTANCE.stopAsync()` to stop the server asynchronously. You can use `.addDoneListener()` on this process to perform additional actions after the server is successfully stopped, such as removing temporary files
```

Comment:
missing full stop at the end of this section

Reply:
Good catch, thanks!